test/integrated/__init__.py

```diff
@@ -0,0 +1,58 @@
+from app_test_case import ChaliceServerThread
+import os
+import time
+import logging
+import importlib.util
+import requests
+from chalice.config import Config as ChaliceConfig
+from azul import config
+
+
+log = logging.getLogger(__name__)
+
+
+class LocalAzulApp:
+    server_thread = None
+
+    @property
+    def base_url(self):
+        """
+        The HTTP endpoint of the locally running Chalice application. Subclasses should use this to derive the URLs
+        for the test requests that they issue.
+        """
+        host, port = self.server_thread.address
+        return f"http://{host}:{port}/"
+
+    _path_to_app = None
+
+    def __init__(self, lambda_name: str):
```

Comment:
Notwithstanding that this will have to be removed (see my summary comment on this PR), this duplicates similar code in an existing test base class. Please be sure to not proliferate copies of existing functionality throughout the code base. Use simple class inheritance, mixins or at least extract common functionality into a reusable function.